torchdata/_torchdata/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+from typing import List, Tuple
+
+# TODO: Add pyi generate script
+class S3Handler:
+    def __init__(self, request_timeout_ms: int, region: str) -> None: ...
+    def s3_read(self, url: str) -> bytes: ...
+    def list_files(self, prefix: str) -> List[str]: ...
+    def set_buffer_size(self, buffer_size: int) -> None: ...
+    def set_multi_part_download(self, multi_part_download: bool) -> None: ...
+    def clear_marker(self) -> None: ...
```

Comment:
Let's remove the pyi file and `_torchdata` folder for now to see if this would help `python` find the library

Reply:
Removed

Reply:
And, for the pyi generation, this should happen during `cmake` time. This feature can be placed into a separate PR.